mesonbuild/wrap/wrap.py

```diff
@@ -217,6 +219,14 @@ def get(self, key: str) -> str:
         except KeyError:
             raise WrapException(f'Missing key {key!r} in {self.basename}')
 
+    def form_hash_file_path(self, subproject_directory: str) -> str:
```

Comment:
I would use `get_hashfile()`

Reply:
Renamed from `form_hash_file_path()` to `get_hashfile()`